concepts/timeline.md

```diff
@@ -6,12 +6,21 @@
 
 Main goal is to learn and sharpen one's knives: programming languages, infrastructure setup, collaboration model setup (matrix, github, etc.)
 
+```{note}
+You go here into a more concrete definition of the work and collaboration: writing software.
+```
 Workflows should be defined and refined, and everyone is encouraged to follow them diligently. Setting this up early allows us to have a much more streamlined workflow naturally in place for the future.
 
 This is also the opportunity to go for random coursera classes, etc.
 
 should already have the brainstorming sessions every week. Need to find a way to properly record them effortlessly, but in a way to gives them utility afterwards (not sure about that: taking minutes is always a burden that nobody wants to shoulder and they are never read afterwards. audio sessions should be informative only (ie: information-sharing), whereas important decisions need to happen in an asynchronous way (more efficient)).
 
+```{note}
+Transcription and LLM summarization models sound like useful here. Can we can get a value-adding workflow by setting up a recording-minutes worflow that integrate them?
+
+Can this be part of a more general 'knowledge base' and 'documentation' effort?
+```
+
```

Comment:
could be, yes. That would also be a nice thing to implement as it would make us "eat our own dog food" (if we are to dabble into AI models), which I believe is very important.